src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +25,11 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void equals_True_Symmetric_X_VV() {
+    RomanNumeral rn2 = new RomanNumeral("VV");
+    assertEquals(rnX, rn2);
```

Comment:
This is a fine test but not of symmetry. Why not add:
```
assertEquals(rn2, rnX);
```

Reply:
For style consistency, why not rename `rn2` to `rnVV`?

Reply:
ok perfect! I will make the changes